src/arm/midr.h

```diff
@@ -219,18 +241,24 @@ inline static uint32_t midr_score_core(uint32_t midr) {
 		case UINT32_C(0x51002110): /* Kryo Silver (Snapdragon 820) */
 		case UINT32_C(0x51002010): /* Kryo Silver (Snapdragon 821) */
 			/* These cores are always in LITTLE core */
-			return 1;
+			return cpuinfo_arm_cpu_perf_level_1;
 		default:
 			/*
 			 * Unknown cores, or cores which do not have big/LITTLE roles.
 			 * To be future-proof w.r.t. cores not yet recognized in cpuinfo, assume position between
 			 * Cortex-A57/A72/A73/A75 and Cortex-A53/A55. Then at least future cores paired with
 			 * one of these known cores will be properly scored.
 			 */
-			return 3;
+			return cpuinfo_arm_cpu_perf_level_3;
 	}
 }
 
+inline static bool cpuinfo_arm_cpu_is_little(uint32_t midr) {
+		cpuinfo_arm_cpu_perf_level l = cpuinfo_arm_cpu_perf_score(midr);
+		return l != cpuinfo_arm_cpu_perf_level_unknown
```

Comment:
So we err towards big core? Meaning if perf level is unknown then it is big core? Also where is the cpuinfo_arm_cpu_perf_level_unknown returned?

Reply:
I haven't changed this logic, just added names. It was returning 3 for unknown cores, see the comment above.